packages/ontario-frontend-cli/core/validation/validatePackageJson.js

```diff
@@ -0,0 +1,42 @@
+const logger = require('../utils/logger');
+const { readPackageJson } = require('../utils/project/readPackageJson');
+
+/**
+ * Checks if a package.json file exists in the given directory.
+ * 
+ * @param {string} dir - The directory to check for package.json.
+ * 
+ * @returns {Promise<boolean>} - Returns a promise that resolves to true if the package.json file exists, false otherwise.
+ */
+async function doesPackageJsonExist(dir = process.cwd()) {
+    try {
+      await readPackageJson(dir);
+      return true;
+    } catch (err) {
+      logger.error(`Error checking for package.json in ${dir}: ${err.message}`);
```

Comment:
Can we make a small change to make the message a bit more relevant?

```suggestion
      logger.error('No package.json found in current directory. Ensure you are performing this command within your Ontario.ca Frontend project');
```

Thoughts? When our process fails for a user, we need to try and provide next steps so they can resolve it - taking the guess work out where we can

Reply:
Agreed, I would just change the word `current` to `target`. It does take the current directory as the default but you can pass in any directory path to the function.